set.mm

```diff
@@ -74469,6 +74501,22 @@ Finite induction (for finite ordinals)
       ( cvv wcel cxp xpexg mp2an ) AEFBEFABGEFCDABEEHI $.
   $}
 
+  ${
+    $( The Cartesian square of a set (see Wikipedia "Cartesian product",
+~ https://en.wikipedia.org/wiki/Cartesian_product#n-ary_Cartesian_power ) is a
+       set.  (Contributed by AV, 13-Jan-2020.) $)
+    sqxpexg $p |- ( A e. V -> ( A X. A ) e. _V ) $=
+      ( wcel cxp cvv xpexg anidms ) ABCAADECAABBFG $.
+  $}
+
+  ${
+    sqxpeqd.1 $e |- ( ph -> A = B ) $.
+    $( Equality deduction for a Cartesian square.  (Contributed by AV,
+       13-Jan-2020.) $)
+    sqxpeqd $p |- ( ph -> ( A X. A ) = ( B X. B ) ) $=
+      ( xpeq12d ) ABCBCDDE $.
+  $}
+
```

Comment:
Isn't basically ~ sqxpeqd an alias for ~ xpeq12d ?

Reply:
In principle yes, but I had 3 reasons to add this theorem:
- it is shorter to use typing a proof (at least with metamath.exe): 2 steps instead of 3
- the proofs will become a little bit shorter: see, for example ~isfin6: minimize will replace ~xpeq12d by ~sqxpeqd, decreasing its proof from 266 to 261 bytes and from 189 to 181 steps (1 essential step).
- I wanted to popularize the term "Cartesian square" (instead of "square Cartesian product"), see also ~sqxpexg.